conda_build/build.py

```diff
@@ -2729,15 +2729,18 @@ def write_test_scripts(metadata, env_vars, py_files, pl_files, lua_files, r_file
                 if utils.on_win:
                     tf.write('set "CONDA_SHLVL=" '
                              '&& @CALL {}\\condabin\\conda_hook.bat {}'
-                             '&& set CONDA_EXE={}'
+                             '&& set CONDA_EXE={python_exe}'
+                             '&& set CONDA_PYTHON_EXE={python_exe}'
+                             '&& set _CE_I={}'
```

Comment:
I want to change this to `_CE_FLAGS`? Is that possible, or will that result in too many other changes?